.env.example

```
@@ -2,3 +2,5 @@ NEXT_PUBLIC_API_URL=
 NEXT_PUBLIC_WS_URL=
 NEXT_PUBLIC_KAKAO_MAP_KEY=
 NEXT_PUBLIC_SITE_URL=
+# MSW 목 서버 사용 여부 (enabled | disabled). 미설정 시 개발 환경에서 enabled
+NEXT_PUBLIC_API_MOCKING=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_API_MOCKING key should go before the NEXT_PUBLIC_API_URL key

(UnorderedKey)